### ISSUE TITLE
HML

### ISSUE THREAD

**Issue by billcolumbia** [billcolumbia/atlas#1]
@joshblack - you have any experience using HML with Sass/CSS?

Working on getting a project up using Webpack and React as a prototype. Might use it for directory. Lmk if you any pointers. Been tearing through old docs and vids, but most stuff is for pre-.13 react and then older webpack.

**Comment by joshblack** [comment]
Hey! And yeah definitely, here's a config I have going right now that works with React 0.14

```js
// webpack.config.js
var path = require('path');
var webpack = require('webpack');

module.exports = {
  devtool: 'eval',
  entry: [
    'webpack-hot-middleware/client',
    './src/index'
  ],
  output: {
    path: path.join(__dirname, 'dist'),
    filename: 'bundle.js',
    publicPath: '/static/'
  },
  plugins: [
    new webpack.HotModuleReplacementPlugin(),
    new webpack.NoErrorsPlugin()
  ],
  module: {
    loaders: [{
      test: /\.js$/,
      loaders: ['babel'],
      include: path.join(__dirname, 'src')
    }, {
      test: /\.css$/,
      loaders: [
        'style',
        'css?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]',
        'postcss'
      ]
    }]
  },
  postcss: [
    require('autoprefixer')
  ]
};
```

```js
// devServer.js
var path = require('path');
var express = require('express');
var webpack = require('webpack');
var config = require('./webpack.config.dev');

var app = express();
var compiler = webpack(config);

app.use(require('webpack-dev-middleware')(compiler, {
  noInfo: true,
  publicPath: config.output.publicPath
}));

app.use(require('webpack-hot-middleware')(compiler));

app.get('*', function(req, res) {
  res.sendFile(path.join(__dirname, 'index.html'));
});

app.listen(3000, 'localhost', function(err) {
  if (err) {
    console.log(err);
    return;
  }

  console.log('Listening at http://localhost:3000');
});
```

And I just run `node devServer.js`

**Comment by billcolumbia** [comment]
Nice thanks duder will play around with this.

**Comment by joshblack** [comment]
No problem! Are you all looking to use React?

**Comment by billcolumbia** [comment]
Maybe. The UI has become very complex and jQuery won't cut it for that kind of state management. I've been trying to learn React, but spent this weekend learning [Vue](https://github.com/vuejs/vue) and I really liked that -- and it was super easy to pick up. (React's details have been hard to follow even with tutorials)

[The Vue stuff I've been playing with (warning it's prototype code](https://github.com/billcolumbia/fieldeditor/tree/master/resources/assets/js)
